src/cljs/nr/new_game.cljs

```diff
@@ -196,7 +196,7 @@
 
 (defn create-new-game [lobby-state user]
   (r/with-let [state (r/atom {:flash-message ""
-                              :format (or (get-in @app-state [:options :default-format]) "standard")
+                              :format (or (get-in @app-state [:options :default-format]) "standard-plus")
```

Comment:
It shouldn't be the default, if we're going to let this run for a while.

Reply:
true - fixed